stellargraph/losses.py

```diff
@@ -26,15 +26,20 @@ def graph_log_likelihood(y_true, y_pred):
     batch_adj = tf.gather(y_true, [0], axis=1)
 
     expected_walks = tf.gather(y_pred, [0], axis=1)
-    sigmoids = tf.gather(y_pred, [1], axis=1)
+    scores = tf.gather(y_pred, [1], axis=1)
 
     adj_mask = tf.cast((batch_adj == 0), "float32")
 
     loss = tf.math.reduce_sum(
         tf.abs(
-            -(expected_walks * tf.math.log(sigmoids + 1e-9))
-            - adj_mask * tf.math.log(1 - sigmoids + 1e-9)
+            -(expected_walks * _log_sigmoid(scores))
```

Comment:
As far as I can tell, this could use https://www.tensorflow.org/api_docs/python/tf/math/log_sigmoid

```suggestion
            -(expected_walks * tf.math.log_sigmoid(scores))
```

Or maybe it has to be:

```suggestion
            (expected_walks * tf.math.log_sigmoid(scores))
```

Reply:
I've added this in 👍